components/brave_wallet/browser/zcash/zcash_orchard_storage.cc

```diff
@@ -0,0 +1,443 @@
+/* Copyright (c) 2024 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/brave_wallet/browser/zcash/zcash_orchard_storage.h"
+
+#include "base/files/file_util.h"
+#include "base/strings/stringprintf.h"
+#include "brave/components/brave_wallet/common/hex_utils.h"
+#include "components/prefs/scoped_user_pref_update.h"
+#include "sql/statement.h"
+#include "sql/transaction.h"
+
+namespace brave_wallet {
+
+namespace {
+constexpr char kNotesTable[] = "notes";
+constexpr char kSpentNotesTable[] = "spent_notes";
+constexpr char kAccountMeta[] = "account_meta";
+
```

Comment:
if we don't want to copy-paste name of tables these could be defined as 
`#define kAccountMeta "account_meta"`
and used as 
```
"CREATE TABLE " kAccountMeta " ("
    "account_id TEXT NOT NULL PRIMARY KEY,"
    "account_birthday INTEGER NOT NULL,"
    "latest_scanned_block INTEGER NOT NULL,"
    "latest_scanned_block_hash TEXT NOT NULL);"
```
so no StringPrintf

Reply:
Done